### ISSUE TITLE
bitping new ~get code

### ISSUE THREAD

**Issue by RocketDevil** [OlivierGaland/CashFactory#60]
new get line for over a month now, pls change in CF :)

docker run -it --mount type=volume,source="bitpingd-volume",target=/root/.bitpingd bitping/bitpingd:latest

and maybe take a look at there site and other os app changes, seems there happend a few changes in all os types...
it was at v0.9.830747851for a long time, and now it is already at v24.1.15.1

so idk how we missed that much updates from that app haha
i mean it makes sense now why there was zero to non income for so long now... :D 

#https://github.com/BitpingApp/Bitping-Node/commit/2e649adfbd2b215e26209b81157604e1a1b3ea9d

**Comment by OlivierGaland** [comment]
Hello,

I was in touch with the change of image and i'm actually testing it since several week before pushing a new release with updated init procedure. Unfortunately I can answer about the close to zero income, even with the new image it stay very low (screen is for more than 1 year running, and current balance for a couple of weeks): 

![image](https://github.com/OlivierGaland/CashFactory/assets/26048157/a0603b7d-1a09-4d6e-820c-37bd41babc6b)

Olivier.

**Comment by RocketDevil** [comment]
yeah since i posted it, i made like 0.00180 (+- 10) so its the low rate of bitping that sucks... 
i mean since i run it (over a year ish) i made $0.198 so far... and thats with 2 nodes at 2 ips started in the same week...

![grafik](https://github.com/OlivierGaland/CashFactory/assets/64044738/6a767eb6-52f4-44c6-b10c-664a43f5f0b0)

**Comment by CoolDouble** [comment]
Hey. Would you mind walking me through on how to get this working? Cheers
